content/blog/2024/craftsperson-and-scientist/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "The craftsperson and the scientist"
+date: 2024-01-17
+subtitle: Two ways of working we've identified through our collaborative practice
+description: "Some people work best iteratively and some would prefer to forge ahead alone – but how do we ensure these two approaches play nicely together in a team?"
+author: honor, emma, kim
+social: social.png
```

Comment:
Think this should be social.jpg or the default site logo just gets pulled in for sharing, unless that is what you want, in which case, go for it https://www.opengraph.xyz/url/https%3A%2F%2F1b3e9967.gfsc.pages.dev%2Fblog%2F2024%2Fcraftsperson-and-scientist%2F

Reply:
Thanks! Fixed

Reply:
nice! LGTM!